helix-core/src/main/java/org/apache/helix/model/LiveInstance.java

```diff
@@ -40,7 +40,20 @@
     LIVE_INSTANCE,
     ZKPROPERTYTRANSFERURL,
     RESOURCE_CAPACITY,
-    CURRENT_TASK_THREAD_POOL_SIZE
+    CURRENT_TASK_THREAD_POOL_SIZE,
+
+    /** Represents {@link LiveInstanceStatus} */
+    STATUS
+  }
+
+  /**
+   * Represents status of a live instance, eg. PAUSED.
+   */
+  public enum LiveInstanceStatus {
+    NORMAL,
```

Comment:
Yeah that's correct. I've thought about it. It is put there to help in the return of java/rest api. If the cluster has fully exited pause, what will the query result look like? "NORMAL" is helping with it. I just wanted to make it as a constant instead of hardcoding in the API.